dkist/dataset/dataset.py

```diff
@@ -192,56 +188,6 @@ def inventory(self):
         """
         return self.meta['inventory']
 
-    """
-    Dataset loading and saving routines.
-    """
-
-    @classmethod
-    def from_directory(cls, directory):
```

Comment:
I think we should probably deprecate these for a while. I know we are technically in beta but we showed them at the workshop and stuff. The implementation of the functions can just call out to the new function and stick an astropy deprecated decorator on top.